### PR TITLE
dns.0.15.1 - via opam-publish

### PR DESCRIPTION
**Pull request by dsheets**
DNS client and server implementation

This is a pure OCaml implementation of the DNS protocol. It is intended to be a
reasonably high-performance implementation, but clarity is preferred rather
than low-level performance hacks.

---
* Homepage: https://github.com/mirage/ocaml-dns
* Source repo: https://github.com/mirage/ocaml-dns.git
* Bug tracker: https://github.com/mirage/ocaml-dns/issues

---
Pull-request generated by opam-publish v0.3.0

### CODE REVIEW

File: packages/dns/dns.0.15.1/opam [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+   LGPL-2.1 with OCaml linking exception &
+   ISC
+         "
+tags: [
+  "org:mirage"
+  "org:xapi-project"
+]
+dev-repo: "https://github.com/mirage/ocaml-dns.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix "--%{base-unix:enable}%-lwt" "--%{mirage-types:enable}%-mirage"]
```
Comment (samoht) [inline packages/dns/dns.0.15.1/opam]: Apparently `--%{mirage-types:enable}%-mirage` should be `--%{mirage-types+tcpip:enable}%-mirage`, see https://travis-ci.org/ocaml/opam-repository/jobs/69355136#L1563

Comment (samoht) [inline packages/dns/dns.0.15.1/opam]: Actually the dependency is not needed, that line https://github.com/mirage/ocaml-dns/blob/master/_oasis#L59 is very suspicious

Comment (samoht) [inline packages/dns/dns.0.15.1/opam]: See https://github.com/mirage/ocaml-dns/pull/79

Comment (dsheets) [inline packages/dns/dns.0.15.1/opam]: I don't know why https://github.com/mirage/ocaml-dns/commit/7612f6b0e078f54c8e97bd2952a79a1ade09108d was added. Also, see `depopt` below. Should be removed?

File: packages/dns/dns.0.15.1/opam [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  "cmdliner"
+  "ipaddr" {>= "2.6.0"}
+  "uri" {>= "1.7.0"}
+  "base64" {>= "2.0.0"}
+  "ounit"
+  "pcap-format" {test}
+  "mirage-profile"
+]
+depopts: [
+  "tcpip"
```
Comment (dsheets) [inline packages/dns/dns.0.15.1/opam]: Not a `depopt`?

Comment (samoht) [inline packages/dns/dns.0.15.1/opam]: true, I'll fix the opam file.

File: packages/dns/dns.0.15.1/opam (hunk 1 above)
Comment (samoht) [inline packages/dns/dns.0.15.1/opam]: No idea. @avsm maybe?

**Pull request merged**